pheweb/serve/server_jeeves.py

```diff
@@ -297,6 +332,9 @@ def get_conditional_regions_for_pheno(self, phenocode, chr, start, end, p_thresh
             print("adding annotations to {} conditional results took {} seconds".format(len(ret), time.time()-t ) )
         return ret
 
+    def get_finemapped_region_boundaries_for_pheno(self, fm_type, phenocode, chrom, start, end):
+        return self.finemapping_dao.get_regions_for_pheno(fm_type, phenocode, chrom, start, end) if self.finemapping_dao is not None else None
```

Comment:
Here I was thinking should we modify factory so that it always constructs dao which returns None if it is not configured to anything? would save a lot of checking

Reply:
Good idea, let's do if we add something